contracts/ToucanCarbonOffsets.sol

```diff
@@ -66,7 +66,7 @@ contract ToucanCarbonOffsets is
 
     /// @dev Returns the current version of the smart contract
     function version() external pure virtual returns (string memory) {
-        return '1.2.0';
+        return '1.3.0';
```

Comment:
Can you include the 1.4-related changes here? Seems they are missing but already merged in master and it would be useful to have them so we stop using a different branch in the toucan-regen contract.

Reply:
oh yes sure! I think I must have just not pulled the latest state :laughing: